validation.js

```diff
@@ -0,0 +1,54 @@
+/* ---------------- contact form variables ----------------*/
+
+const form = document.querySelector('.contact-form');
+const emailRegex = /^([a-z|\d])+@+([a-z]){2,}\.+([a-z)]{2,})/;
+const emailInput = document.getElementById('email');
+const nameInput = document.getElementById('name');
+const msgInput = document.getElementById('message');
+const emailErrorMessage = document.querySelector('.email-error');
+const formErrorMessage = document.querySelector('.form-error');
+const nameErrorMessage = document.querySelector('.name-error');
+const messageErrorMessage = document.querySelector('.message-error');
+
+/* ---------- Contact Form -------------- */
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (nameInput.value !== null && msgInput.value !== null && emailRegex.test(emailInput.value)) {
+    form.submit();
+    formErrorMessage.textContent = 'Succesfully sent';
+    formErrorMessage.style.color = 'green';
+    emailErrorMessage.style.display = 'none';
+    nameErrorMessage.style.display = 'none';
+    messageErrorMessage.style.display = 'none';
+  } else {
+    formErrorMessage.textContent = 'the form was not sent';
+
+    if (!emailInput.value) {
+      emailErrorMessage.textContent = 'email is required';
+      emailErrorMessage.style.display = 'block';
+    } else if (!emailRegex.test(emailInput.value)) {
+      emailErrorMessage.textContent = 'the email input should follow this format : xyz@exe.com';
```

Comment:
[OPTIONAL]
An error message like xyz@exe.com might confuse some people. To make it more user-friendly, you can replace the error message with john@gmail.com or something user-friendly.